node/testing/src/scenarios/p2p/basic_connection_handling.rs

```diff
@@ -83,7 +83,7 @@ impl SimultaneousConnections {
     }
 }
 
-/// Connections between all peers are symmetric, i.e. iff the node1 has the node2 among its active peers, then the node2 should have the node1 as its active peers.
+/// Connections between all peers are symmetric, i.e. if the node1 has the node2 among its active peers, then the node2 should have the node1 as its active peers.
```

Comment:
You may need to check with whoever wrote the original comment, but I don't think these `iff`s were typos https://en.wikipedia.org/wiki/If_and_only_if